packages/components/Shimmer/src/Shimmer.tsx

```diff
@@ -0,0 +1,105 @@
+/** @jsx withSlots */
+import {
+  compose,
+  UseSlots,
+  buildProps,
+  mergeProps,
+  withSlots
+} from '@fluentui-react-native/experimental-framework';
```

Comment:
Just a heads up, Morse is renaming this so you might hit merge issues with the name